json_spec/README.md

```diff
@@ -1,16 +1,2589 @@
+        <td>string, number, boolean</td>
+        <td>Validates that a <em>measurement</em> and a comparison value are not equal.</td>
+    </tr>
+    <tr>
+        <td><strong>LESS_THAN</strong></td>
+        <td>number</td>
+        <td>number</td>
+        <td>Validates that a <em>measurement</em> is less than a comparison value.</td>
+    </tr>
+    <tr>
+        <td><strong>LESS_THAN_OR_EQUAL</strong></td>
+        <td>number</td>
+        <td>number</td>
+        <td>Validates that a <em>measurement</em> is less than or equal to a comparison value.</td>
+    </tr>
+    <tr>
+        <td><strong>GREATER_THAN</strong></td>
+        <td>number</td>
+        <td>number</td>
+        <td>Validates that a <em>measurement</em> is greater than a comparison value.</td>
+    </tr>
+    <tr>
+        <td><strong>GREATER_THAN_OR_EQUAL</strong></td>
+        <td>number</td>
+        <td>number</td>
+        <td>Validates that <em>measurement</em> is greater than or equal to a comparison value.</td>
+    </tr>
+    <tr>
+        <td><strong>REGEX_MATCH</strong></td>
+        <td>string</td>
+        <td>string, array[string]</td>
+        <td>Validates that a <em>measurement</em> is found to match one or more regex strings contained by the <em>validator</em>.</td>
+    </tr>
+    <tr>
+        <td><strong>REGEX_NO_MATCH</strong></td>
+        <td>string</td>
+        <td>string, array[string]</td>
+        <td>Validates that a <em>measurement</em> does not match any regex strings contained by the <em>validator</em>.</td>
+    </tr>
+    <tr>
+        <td><strong>IN_SET</strong></td>
+        <td>string,number</td>
+        <td>array[string], array[number]</td>
+        <td>Validates that a <em>measurement</em> exists within a set of allowable values contained by the <em>validator</em>.</td>
+    </tr>
+    <tr>
+        <td><strong>NOT_IN_SET</strong></td>
+        <td>string, number</td>
+        <td>array[string], array[number].</td>
+        <td>Validates that a <em>measurement</em> does not exist within a set of allowables values contained by the <em>validator</em>.</td>
+    </tr>
+</table>
+
+##### Example
+
+```json
+{
+  "testStepArtifact": {
+    "measurement": {
+      "name": "measured-fan-speed-100",
+      "unit": "RPM",
+      "hardwareInfoId": "5",
+      "subcomponent": {
+        "name": "FAN1",
+        "location": "F0_1",
+        "version": "1",
+        "revision": "1",
+        "type": "UNSPECIFIED"
+      },
+      "validators": [
+        {
+          "name": "80mm_fan_upper_limit",
+          "type": "LESS_THAN_OR_EQUAL",
+          "value": 11000.0
+        },
+        {
+          "name": "80mm_fan_lower_limit",
+          "type": "GREATER_THAN_OR_EQUAL",
+          "value": 8000.0
+        }
+      ],
+      "value": 100221.0
+    },
+    "testStepId": "300"
+  },
+  "sequenceNumber": 400,
+  "timestamp": "2022-07-26T02:16:10.393209Z"
+}
+```
+
+
+## <a name="contributors">Specification Contributors</a>
```

Comment:
Maybe remove this whole section now since it's a living MD doc, so we'll have git history?

Reply:
i can, it wouldnt cover people that contributed to this but didnt have commits though; maybe a compromise is to keep it in the pdf and remove it here

Reply:
done. please check/resolve